src/components/shared/marginals/events.jsx

```diff
@@ -0,0 +1,65 @@
+import { Heading, Text } from '../typography/index.jsx';
+import eventData from '../../../data/eventData.js';
+
+export default function Event() {
+	return (
+		<div className='p-8' style={{ border: '1px solid red' }}>
+			<Heading
+				variant='h1'
+				className='text-primary-yellow text-center'
+				style={{
+					fontSize: '80px',
+					textShadow: '4.608px 4.608px 0px #000',
+					WebkitTextStrokeWidth: 2.71052622795105,
+					WebkitTextStrokeColor: '#252525',
+				}}>
```

Comment:
use variants

Reply:
as variant using h1

Reply:
Another variant could be added the required styles in Heading component.